src/dosbox.cpp

```diff
@@ -964,23 +964,18 @@ void DOSBOX_Init(void) {
 	Pbool->Set_help("Enable ipx over UDP/IP emulation.");
 #endif
 
-#if C_NE2000
-	secprop=control->AddSection_prop("ne2000",&NE2K_Init,true);
-	MSG_Add("NE2000_CONFIGFILE_HELP",
-		"macaddr -- The physical address the emulator will use on your network.\n"
-		"           If you have multiple DOSBoxes running on your network,\n"
-		"           this has to be changed. Modify the last three number blocks.\n"
-		"           I.e. AC:DE:48:88:99:AB.\n"
-		"realnic -- Specifies which of your network interfaces is used.\n"
-		"           Write \'list\' here to see the list of devices in the\n"
-		"           Status Window. Then make your choice and put either the\n"
-		"           interface number (2 or something) or a part of your adapters\n"
-		"           name, e.g. VIA here.\n"
-
-	);
+#if C_SLIRP
+	secprop = control->AddSection_prop("ethernet", &NE2K_Init, true);
 
 	Pbool = secprop->Add_bool("ne2000", when_idle,  true);
-	Pbool->Set_help("Enable Ethernet passthrough. Requires [Win]Pcap.");
+	Pbool->Set_help("Emulate a Novell NE2000 network card on a software-based\n"
+	                "network (using libSLIRP) with properties as follows:\n"
+	                " - 10.0.2.0/24   : Virtual LAN\n"
+	                " - 10.0.2.2      : Gateway and DHCP service\n"
+	                " - 10.0.2.3      : Virtual DNS server\n"
+	                " - 10.0.2.15     : First IP provided by DHCP (your IP)\n"
+	                " - 255.255.255.0 : Subnet mask\n"
```

Comment:
Both CIDR notation and subnet mask seem redundant. Perhaps consider dropping one? Or have you got it here for those who know one but not the other?

Also, I don't know how slirp works in this regard, but is there a reason why DNS is on a different IP address to gateway services? Could they all be on the same address (like most home routers do)?

Reply:
Thanks for reviewing @shermp.

> Both CIDR notation and subnet mask seem redundant. Perhaps consider dropping one? Or have you got it here for those who know one but not the other?

Yes; the latter reason.  Most (all?) DOS and Win31 network tools use subnet mask - so we definitely want to keep `255.255.255.0`.

At the same time, I was hesitant to remove the `/24` from the LAN space, given a bare `10.0.2.0` appears like an IP (and I don't want users accidentally using it).  That said - this purely informational, so perhaps it's best to remove it.

Maybe there's a better layout or way to describe these, if you can suggest it?

Reply:
> Also, I don't know how slirp works in this regard, but is there a reason why DNS is on a different IP address to gateway services? Could they all be on the same address (like most home routers do)?

They could be combined, but this differs from slirp's default configuration (which you'll find in PCem, 86box, and DOSBox-X).  So I want to use those canned setting as-is.  This offers package-integrators a trusted configuration that they could hardcode values in `.ini` and config files - and know that it will 'just work' in Staging.

Reply:
> > Also, I don't know how slirp works in this regard, but is there a reason why DNS is on a different IP address to gateway services? Could they all be on the same address (like most home routers do)?
> 
> They could be combined, but they would then differs from slirp's default configuration (which you'll find in PCem, 86box, and DOSBox-X).  So I want to use those canned default values as-is.  
> 
> This lets package integrators "trust" these settings in so much as they can setup a tcp/ip stack, configuring it, install games or programs - and share that integrated package for less knowledgeable users who can run it straight away.

If that's what everyone else is doing, who am I to suggest otherwise :)

Reply:
> Thanks for reviewing @shermp.
> 
> > Both CIDR notation and subnet mask seem redundant. Perhaps consider dropping one? Or have you got it here for those who know one but not the other?
> 
> Yes; the latter reason.  Most (all?) DOS and Win31 network tools use subnet mask - so we definitely want to keep `255.255.255.0`.
> 
> At the same time, I was hesitant to remove the `/24` from the LAN space, given a bare `10.0.2.0` appears like an IP (and I don't want users accidentally using it).  That said - this purely informational, so perhaps it's best to remove it.
> 
> Maybe there's a better layout or way to describe these, if you can suggest it?

Can't think of a better way to handle this, so leave as is I suppose.